packages/steiger-plugin-fsd/src/forbidden-imports/README.md

```diff
@@ -172,3 +172,12 @@ flowchart BT
 ## Rationale
 
 This is one of the main rules of Feature-Sliced Design, it ensures low coupling and predictability in refactoring.
+
+## Related Rules
```

Comment:
suggestion (non-blocking): I think we can make this part stand out more by moving it up, say, right before the example. We can also use [GitHub's alert syntax](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax#alerts) to convert this block into a little callout (type "Note").